include/deal.II/lac/dynamic_sparsity_pattern.h

```diff
@@ -0,0 +1,538 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2015 - 2015 by the deal.II authors
```

Comment:
Wrong year -- much of this is much older than 2015. See my upcoming comment on using git mv.